vision/cloud-client/snippets.py

```diff
@@ -0,0 +1,339 @@
+#!/usr/bin/env python
+
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import io
+import os
+
+# Imports the Google Cloud client library
+from google.cloud import vision
+
+
+def detect_faces(path):
+    # Instantiates a client
+    vision_client = vision.Client()
+
+    # Loads the image into memory
+    with io.open(path, 'rb') as image_file:
+        content = image_file.read()
+        image = vision_client.image(
+            content=content)
+
+    # Performs face detection on the image file
```

Comment:
This comment is obvious in context.

(Typically focus comments on *why* over *what*, especially if this isn't a quickstart)